.circleci/config.yml

```
@@ -499,6 +526,17 @@ jobs:
           paths:
             - changed-files
 
+  validate-locales-only:
+    executor: node-browsers-small
+    steps:
+      - run: *shallow-git-clone
+      - run: sudo corepack enable
+      - attach_workspace:
+          at: .
+      - run:
+          name:
+          command: yarn tsx .circleci/scripts/validate-locales-only.ts
```

If you don't give the command a name, it should just be one line like this:
```suggestion
      - run: yarn tsx .circleci/scripts/validate-locales-only.ts
```